scene/resources/texture.cpp

```diff
@@ -29,7 +29,6 @@
 /*************************************************************************/
 
 #include "texture.h"
-
```

Comment:
Please keep this line separating the definition header from the rest. See http://docs.godotengine.org/en/latest/community/contributing/code_style_guidelines.html#header-includes

Reply:
fixed